src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -221,22 +173,33 @@ public boolean runUntilEnd() throws IOException, InterruptedException {
         return processResult == 0;
     }
 
+    boolean useLLVM = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5574548)